template/v1/Dockerfile

```diff
@@ -49,6 +49,11 @@ RUN echo "source /usr/local/bin/_activate_current_env.sh" | tee --append /etc/pr
 RUN mkdir -p /opt/amazon/sagemaker/sagemaker-code-editor-server-data /opt/amazon/sagemaker/sagemaker-code-editor-user-data \
     && chown $MAMBA_USER:$MAMBA_USER /opt/amazon/sagemaker/sagemaker-code-editor-server-data /opt/amazon/sagemaker/sagemaker-code-editor-user-data
 
+# SQLEditor - create dir to store autocomplete schema files
+RUN mkdir -p /opt/amazon/sagemaker/sql-autocomplete-schema \
```

Comment:
Can we re-structure to `/opt/amazon/sagemaker/sagemaker-sql-experience-user-data/sql-autocomplete-schema` and give ownership on `/opt/amazon/sagemaker/sagemaker-sql-experience-user-data` directory, 
so that our extension can use this directory for all future use-cases?

Reply:
Updated. Please check